lib/versioner/staggered_test.go

```diff
@@ -70,7 +70,19 @@ func TestStaggeredVersioningVersionCount(t *testing.T) {
 
 	<-v.testCleanDone
 
-	rem := v.toRemove(files, now)
+	versionsWithMtime := make([]versionWithMtime, 0, len(files))
+	for _, file := range files {
+		versionTime, err := time.ParseInLocation(TimeFormat, ExtractTag(file), loc)
+		if err != nil {
+			t.Fatal(err)
+		}
+		versionsWithMtime = append(versionsWithMtime, versionWithMtime{
+			name:  file,
+			mtime: versionTime,
+		})
+	}
+
+	rem := v.toRemove(versionsWithMtime, now)
```

Comment:
This test should be rewritten. There is no need to run the versioner at all to just call toRemove, and it's now quite confusing that we extract the mtime from the time when we explicitly don't act like that on the info in the tag. (Sorry for pushing back on this; it's not your fault the test is what it is, but this makes it worse.)